### ISSUE TITLE
i am trying to fetch novel from lightnovelpub and lightnovelworld

### ISSUE THREAD

**Issue by baqirNemer** [dipu-bd/lightnovel-crawler#2373]
<img width="1440" alt="Screenshot 2024-05-12 at 05 04 46" src="https://github.com/dipu-bd/lightnovel-crawler/assets/88574788/557f0a95-7f0f-4c5a-b51f-76dd3fa3e8e7">
each time error -9 occurs I am chrome opens automatically and the download stops 
[meta.json](https://github.com/dipu-bd/lightnovel-crawler/files/15284827/meta.json)
and I get this JSON file in the directory where the novel is supposed to be downloaded

**Comment by jango1985** [comment]
I just tried to download a novel from lightnovelworld and it downloaded a little then had a cloudfare captcha in chromium browser it opened.  I clicked the checkbox manually and it would download about 16 or so chapters before asking again then I got the bright idea to look for a free captcha solver extension and load it into the lncrawler chromium.  I found this https://github.com/NopeCHALLC/nopecha-extension and loaded it into the browser in developer mode.  Now it auto clicks the verify your human checkbox for me.  Not sure if this will have to be redone next time lncrawler tries to download another book through chromium but even it does still more easier than constantly clicking the captcha box myself.
